<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlBlockTypeSelectView</class>
 <widget class="QWidget" name="ControlBlockTypeSelectView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="horizontalSpacing">
    <number>10</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widgetTree" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>1</number>
      </property>
      <item row="0" column="0">
       <widget class="TreeView" name="treeView">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="autoScrollMargin">
         <number>8</number>
        </property>
        <property name="indentation">
         <number>0</number>
        </property>
        <property name="headerHidden">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="widgetTable" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="margin">
       <number>1</number>
      </property>
      <item row="0" column="0">
       <widget class="TableView" name="tableView">
        <property name="minimumSize">
         <size>
          <width>480</width>
          <height>0</height>
         </size>
        </property>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TreeView</class>
   <extends>QTreeView</extends>
   <header>TypeItemView.h</header>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>TypeItemView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
